test_themes.py

```
@@ -0,0 +1,96 @@
+
+
+def test_mafia_theme_same_level_count_as_military():
+    assert len(MAFIA_RANKS) == len(RANKS)
+
+
+def test_validate_theme_rejects_wrong_count():
+    broken = [dict(r) for r in MAFIA_RANKS]
+    broken[0]["count"] += 1
+    try:
+        validate_theme(broken, MAFIA_FLAG)
+        assert False, "expected ValueError"
```

Ruff (0.16.1)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)